src/cljs/rems/fields.cljs

```diff
@@ -205,19 +205,19 @@
                 [:label.form-check-label {:for option-id}
                  (localized label)]])))]))
 
-;; TODO: custom :diff-component, for example link to both old and new attachment
 (defn attachment-field
   [{:keys [validation app-id on-change on-set-attachment on-remove-attachment] :as opts}]
   (let [id (:field/id opts)
         title (localized (:field/title opts))
         value (:field/value opts)
         filename (get-in opts [:field/attachment :attachment/filename])
         click-upload (fn [e] (when-not (:readonly opts) (.click (.getElementById js/document (id-to-name id)))))
-        filename-field [:div.field
-                        [:a.btn.btn-secondary.mr-2
-                         {:href (str "/api/applications/attachment/" value)
-                          :target :_new}
-                         filename " " (external-link)]]
+        link (fn [attachment-id filename]
+               [:div.field
+                [:a.btn.btn-secondary.mr-2
+                 {:href (str "/api/applications/attachment/" attachment-id)
+                  :target :_new}
```

Comment:
Should be _blank https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-target